docs/examples/pytorch/single_stage_detector/pytorch_ssd.rst

```diff
@@ -0,0 +1,76 @@
+Single Shot MultiBox Detector training in PyTorch
+============================
+
+This example implements Single Shot Multibox Detector originally published by Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy, 
+Scott Reed, Cheng-Yang Fu, Alexander C. Berg as `SSD: Single Shot MultiBox Detector <https://arxiv.org/abs/1512.02325>`_.
+
+Code is based on MLPerf example from `here <https://github.com/mlperf/training/tree/master/single_stage_detector/ssd>`_ and has been modified to use DALI. 
+
+To run use following command:
+
+.. code-block:: bash
+
+   python train.py
```

Comment:
Could we somehow add also info, how to lunch it in distributed training manner? Like
`python -m torch.distributed.launch --nproc_per_node=8 train.py`